src/main/resources/org/jenkinsci/plugins/workflow/flow/GlobalDefaultFlowDurabilityLevel/help-durabilityHint.html

```diff
@@ -1,20 +1,27 @@
 <div>
     <p>
-        This setting provides a trade-off between how fast pipelines run and their "durability", or ability to survive unexpected failures while running.
+        This setting allows users to change the default durability mode for running Pipelines.  In most cases this is a
+        trade-off between performance and the ability for running pipelines to resume after unplanned Jenkins outages.
 
-        <br><strong>Facts:</strong>
+        <br><br><strong>What does this do?</strong>
         <ul>
-            <li>The lowest durability modes (suggested) can be MUCH faster and write to disk FAR less often.</li>
-            <li>All pipelines persist their data when they complete/fail, are paused, or the master restarts gracefully.</li>
-            <li>Running pipelines that do NOT persist data with every step (lower durability settings) may lose data if Jenkins is not shut down gracefully.</li>
-            <li>Pipelines may not be able to resume or be visualized if they cannot persist data.</li>
-            <li>If a running pipeline loses data, you can always run the pipeline again.</li>
+            <li>Previously, running pipelines wrote data constantly, so that they could resume even if Jenkins fails.
+                This setting gives the user the ability to adjust this write behavior.</li>
+            <li>Higher-performance/lower-durability modes write data to disk <strong>much</strong> less often for running pipelines.</li>
+            <li>Writing data less often can massively reduce build times for Pipelines with many steps or complex logic.
+                For pipelines which spend most of their time waiting for a shell/batch script to run, the difference will be less visible.</li>
+            <li>Running pipelines with lower durability settings may lose data <strong>if</strong> they do not finish and Jenkins is not shut down gracefully:</li>
+                <ul>
+                    <li>A "graceful" shutdown where Jenkins goes through a full shutdown process, such as visiting <code>http://[jenkins-server]/exit</code></li>
+                    <li>A "dirty" shutdown, such as using <code>kill -9</code> to terminate the Jenkins process, may prevent incomplete pipelines from persisting data</li>
```

Comment:
Note that `SIGTERM` might be graceful or dirty, depending on the servlet container.

Reply:
I agree it needs to be mentioned, but this isn't the place -- too much overcrowding and a fairly small detail.

The place for that is more detailed documentation that will go on Jenkins.io, where I plan to explain in more depth how to guarantee clean shutdowns and what may cause dirty ones.